### PR TITLE
Run3-hc362 Define the modified scenarios for 2024 (Affecting ZDC geometry)

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Define the modified scenarios for 2024 (Affecting ZDC geometry)

#### PR validation:

Use the runTheMatrix test workflows 

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

To be backported to CMSSW_14_1_X once #46011 is integrated

### CODE REVIEW

**Comment by perrotta** [comment]
please test

**Comment by mandrenguyen** [comment]
urgent
Needed to build 14_1_0 for heavy-ion data taking

**Comment by mmusich** [comment]
> -1

@bsunanda  you'll need to change the customization function introduced at https://github.com/cms-sw/cmssw/pull/45511 to remove the `Exclude` parameter.

https://github.com/cms-sw/cmssw/blob/79c1e019742c72870eacc84f341172e7d70f368a/HLTrigger/Configuration/python/customizeHLTforCMSSW.py#L52-L61

(incidentally, that's also why I was suggesting you privately to load a configuration fragment instead of the fully specified configuration... )

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 1]
```diff
@@ -49,12 +49,11 @@ def customiseForOffline(process):
     return process
 
 # Adding zdc Topology producer
-def customizeHLTfor45511(process):
+def customizeHLTfor46033(process):
     """Add topology producer for ZDC"""
 
     if hasattr(process, 'ZdcGeometryFromDBEP'):
         process.ZdcTopologyEP = cms.ESProducer( "ZdcTopologyEP",
-                                                Exclude = cms.untracked.string( "" ),
```
Comment (mmusich) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py:57]: I would suggest to use instead:

```python
# Adding zdc Topology producer
def customizeHLTfor46033(process):
    """Add topology producer for ZDC"""

    for esprod in esproducers_by_type(process, 'ZdcGeometryFromDBEP'):
        process.load("Geometry.ForwardGeometry.zdcTopologyEP_cfi")
        break

    return process
```

to be more future-proof...

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by mandrenguyen** [comment]
@cms-sw/hlt-l2 @cms-sw/alca-l2 @cms-sw/db-l2 @cms-sw/upgrade-l2 please have a look

**Comment by mmusich** [comment]
+hlt

**Comment by perrotta** [comment]
+1

**Comment by bsunanda** [comment]
@srimanob Please approve this

**Comment by srimanob** [comment]
+Upgrade

**Comment by mandrenguyen** [comment]
+1

**Comment by bsunanda** [comment]
Yes I just submitted the backport. The next thing is to create the payloads. I shall do that tomorrow when I can get the IB with this merged PR

**Pull request merged**